docset/winserver2012-ps/dnsserver/Get-DnsServer.md

```diff
@@ -22,23 +22,22 @@ Get-DnsServer [-AsJob] [-CimSession <CimSession[]>] [-ComputerName <String>] [-T
 
 ## DESCRIPTION
 The **Get-DnsServer** cmdlet retrieves a Domain Name System (DNS) server configuration.
-The DNS server must be running Windows Server® 2008 R2 operating system or above.
 
 You can pipe the output of the **Get-DnsServer** cmdlet to the **Export-Clixml** cmdlet, which generates an XML file of the configuration.
 You can use the XML file to back up or transfer DNS settings between computers.
-For more information about **Export-Clixml**, see Using the Export-Clixml Cmdlethttp://technet.microsoft.com/en-us/library/ee176824.aspx.
+For more information about **Export-Clixml**, see [Using the Export-Clixml cmdlet](https://technet.microsoft.com/library/ee176824.aspx).
```

Comment:
Please update the URL to https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/export-clixml